package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "karma start --single-run",
-    "dist": "mkdirp dist && browserify index.js -o dist/buildium-angular-ui.js",
+    "compile": "babel src --out-dir lib/ --presets=es2015",
+    "dist": "mkdirp dist && browserify lib/index.js -o dist/buildium-angular-ui.js",
     "docs": "node .build/docs.js",
-    "prepublish": "npm run dist",
+    "prepublish": "npm run compile && npm run dist",
     "version": "npm run docs -- --gh-pages"
```

Comment:
Are you missing the "build" script? https://github.com/buildium/angular-redux/pull/6/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R10